setup.py

```diff
@@ -1,12 +1,16 @@
 from setuptools import setup
 
+#bring in __version__ from source
+#per https://stackoverflow.com/a/17626524
+#and https://stackoverflow.com/a/2073599
+with open('lineman/version.py') as ver: exec(ver.read())
```

Comment:
ohhh okay. Yeah this seems fine. Even though it has an exec we are doing very minimal stuff so it isn't a big deal